tests/Client/UpdaterTest.php

```diff
@@ -314,6 +315,262 @@ public function providerVerifiedDelegatedDownload(): array
                     'level_3_below_terminated' => 1,
                 ],
             ],
+            // A terminating role only has an effect if the target path matches
+            // the role, otherwise the role is not evaluated.
+            // Roles after terminating delegation where the target path does match not
+            // the terminating role are evaluated.
```

Comment:
"...where the target path does not match ARE evaluated"? Shouldn't this be "are not"?

Reply:
fixed